ush/parsing_namelists_FV3.sh

```diff
@@ -631,6 +631,24 @@ EOF
   use_zmtnblck = ${use_zmtnblck:-".true."}
 EOF
   fi
+  
+  if [[ "${DO_OCN_SPPT:-NO}" == "YES" ]]; then
+    cat >> input.nml <<EOF
+  OCNSPPT=${OCNSPPT:-1.0}
+  OCNSPPT_LSCALE=${OCNSPPT_LSCALE:-500e3}
+  OCNSPPT_TAU=${OCNSPPT_TAU:-21600}
```

Comment:
The `defaults` here will never be realized if they are defined in `config.efcs`, so we should probably set these as:
```suggestion
  OCNSPPT=${OCNSPPT}
  OCNSPPT_LSCALE=${OCNSPPT_LSCALE}
  OCNSPPT_TAU=${OCNSPPT_TAU}
```